.gitignore

```diff
@@ -1,170 +1,7 @@
-# Byte-compiled / optimized / DLL files
+# ignore python cache directory
 __pycache__/
-*.py[cod]
-*$py.class
 
-# C extensions
-*.so
+# ignore poetry lock file - created automatically
+poetry.lock
 
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-share/python-wheels/
-*.egg-info/
-.installed.cfg
-*.egg
-MANIFEST
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.nox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*.cover
-*.py,cover
-.hypothesis/
-.pytest_cache/
-cover/
-pytest-coverage.txt
-pytest.xml
```

Comment:
All of these are good to have because [the repo has unit tests](https://github.com/vesper-arch/Slay-the-Spire-in-Python/tree/main/tests)